domains/tejaswa.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FirE-A",
+        "email": "tejasgamer098@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://tejas-wa.github.io/website/"
```

Comment:
Invalid CNAME record, can not contain `https://` or `/` (maybe you wwanted to use a URL record?)